### ISSUE TITLE
Wrong index onSelect BottomBarBubble() 

### ISSUE THREAD

**Issue by KiloNovemberDelta** [tuannvm2109/bottom_bar_matu#4]
Hello,
Thank you for this open source package! 

I tried the example, adding only "controller.jumpToPage(index);" in the onSelect function. However, the navigation seems broken, the page displayed are not the right one. I use the version 1.4.0. 
Here is the minimal code : 

```
import 'package:flutter/material.dart';

import 'package:bottom_bar_matu/bottom_bar_matu.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: const MyHomePage(title: 'Flutter Bottom Bar Bubble'),
    );
  }
}

class MyHomePage extends StatefulWidget {
  const MyHomePage({Key? key, required this.title}) : super(key: key);

  final String title;

  @override
  State<MyHomePage> createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  final PageController controller = PageController();

  @override
  void dispose() {
    super.dispose();
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(title: Text(widget.title)),
      bottomNavigationBar: BottomBarBubble(
        items: [
          BottomBarItem(
            iconData: Icons.home,
            // label: 'Home',
          ),
          BottomBarItem(
            iconData: Icons.chat,
            // label: 'Chat',
          ),
          BottomBarItem(
            iconData: Icons.notifications,
            // label: 'Notification',
          ),
          BottomBarItem(
            iconData: Icons.calendar_month,
            // label: 'Calendar',
          ),
          BottomBarItem(
            iconData: Icons.settings,
            // label: 'Setting',
          ),
        ],
        onSelect: (index) {
          // implement your select function here
          print(index); // Here, index is not write at all.. 
          controller.jumpToPage(index);
        },
      ),
      body: PageView(
        controller: controller,
        children: const <Widget>[
          Center(
            child: Text('First Page'),
          ),
          Center(
            child: Text('Second Page'),
          ),
          Center(
            child: Text('Third Page'),
          ),
          Center(
            child: Text('Four Page'),
          ),
          Center(
            child: Text('Five Page'),
          ),
        ],
      ),
    );
  }
}`
```

**Comment by tuannvm2109** [comment]
Thank you for your report. It was my mistake recently when I updated `onSelect` function. I have already published a new version, 1.5.0 to fix this issue.

**Comment by KiloNovemberDelta** [comment]
Yes it is fixed now, thank you!
